site/docs/v1/tech/apis/_user-consent-request-body.adoc

```diff
@@ -16,8 +22,8 @@ ifdef::update[]
 [field]#userConsent.status# [type]#[String]# [required]#Required#::
 The status of the User consent. Possible values are:
 +
-- `Active`
-- `Revoked`
+ - `Active`
```

Comment:
Why the spaces?

Reply:
Because without them the block of text following the list is rendered as part of the second list element
![image](https://user-images.githubusercontent.com/3409780/212114010-dca744df-1d0e-45f7-bd10-3f9b146c1a72.png)

The spaces make the block align with "The status of.." block above the list.